lib/query_helper/sql_sort.rb

```diff
@@ -41,12 +42,12 @@ def attributes_sql_expression(sort_attribute)
           if direction == "desc"
             case ActiveRecord::Base.connection.adapter_name
             when "SQLite" # SQLite is used in the test suite
-              direction = "desc"
+              direction = @keep_nulls_last ? "desc nulls last" : "desc"
             else
               direction = "desc nulls last"
```

Comment:
do we need @keep_nulls_last condition here as well?

Reply:
@karanmodi19 No. Because here we are already adding `nulls last` to `desc`.